metis-harvesting/src/main/java/eu/europeana/metis/harvesting/oaipmh/OaiRecordParser.java

```diff
@@ -45,26 +48,25 @@
           "/*[local-name()='metadata']" +
           "/child::*";
 
-  private static final String IS_DELETED_XPATH = "string(/*[local-name()='OAI-PMH']" +
+  private static final String HEADER_XPATH = "/*[local-name()='OAI-PMH']" +
           "/*[local-name()='GetRecord']" +
           "/*[local-name()='record']" +
-          "/*[local-name()='header']" +
-          "/@status)";
+          "/*[local-name()='header']";
```

Comment:
Should we compute the constant in one go, removing the "+"?
(Same for the to other xpath)

Reply:
Okay makes sense